core/src/main/java/org/apache/calcite/jdbc/SimpleCalciteSchema.java

```diff
@@ -100,36 +100,8 @@ private SimpleCalciteSchema(@Nullable CalciteSchema parent,
     return null;
   }
 
-  @Override protected @Nullable CalciteSchema getImplicitSubSchema(String schemaName,
-      boolean caseSensitive) {
-    // Check implicit schemas.
-    final String schemaName2 =
-        caseSensitive ? schemaName
-            : caseInsensitiveLookup(schema.getSubSchemaNames(), schemaName);
-    if (schemaName2 == null) {
-      return null;
-    }
-    final Schema s = schema.getSubSchema(schemaName2);
-    if (s == null) {
-      return null;
-    }
-    return new SimpleCalciteSchema(this, s, schemaName2);
-  }
-
-  @Override protected @Nullable TableEntry getImplicitTable(String tableName,
```

Comment:
what happened to all these functions?
(I don't know what they were meant for.)

Reply:
`SimpleCalciteSchema` and `CachingCalciteSchema` formerly had two list for tables and sub schemas each:
* implicit entries which are loaded from the associated Schema (member `schema`)
* explicit entries which are managed by calls to `add...`

The concatenation of both lists was formerly happening in `CalciteSchema.getTableNames`.  This is now happening in `CalciteSchema.tables()` with the help of `Lookup.concat()`.
There is no longer a special need to have different implementations of `getImplicitTables` for `SimpleCalciteSchema` and `CachingCalciteSchema` since the caching is done in `CachingCalciteSchema.enhanceLookup` for both tables and sub schemas.